test/debate.js

```diff
@@ -90,6 +99,236 @@ describe('Debate test', () => {
+                    questionObj.answers[1].should.equal('No');
+                    done();
+                });
+
+                debate.sendNewQuestion(new debate.Question('Does this test work ?', ['Yes', 'No']));
+            });
+        });
+
+        describe('answerQuestion', () => {
+            it('valid response', (done) => {
+                client.on('newQuestion', (questionObj) => {
+                    client.emit('answerQuestion', {questionId : questionObj.id, answerId : 0}, (res) => {
+                        res.should.equal(true);
+                        done();
+                    });
+                });
+
+                debate.sendNewQuestion(new debate.Question('Does this test work ?', ['Yes', 'No']));
+            });
+
+            it('invalid questionID', (done) => {
+                client.on('newQuestion', (questionObj) => {
+                    client.emit('answerQuestion', {questionId : -1, answerId : 1}, (res) => {
+                        res.should.equal(false);
+                        done();
+                    });
+                });
+
+                debate.sendNewQuestion(new debate.Question('Does this test work ?', ['Yes', 'No']));
+            });
+
+            it('invalid object', (done) => {
+                client.on('newQuestion', (questionObj) => {
+                    client.emit('answerQuestion', {myFieldIsInvalid: 12}, (res) => {
+                        res.should.equal(false);
+                        done();
+                    });
+                });
+
+                debate.sendNewQuestion(new debate.Question('Does this test work ?', ['Yes', 'No']));
+            });
+        });
+
+        afterEach(() => {
+            client.close();
+        });
+
+        after(() => {
+            admin.close();
+        });
+    });
+
+    describe("Debate admin functions", () => {
+        let client;
+        let id;
+        let debate;
+
+        before((done) => {
+            admin = io.connect(`http://localhost:${SocketConfig.SOCKET_PORT}${SocketConfig.ADMIN_NAMESPACE}`, {
+                path: SocketConfig.DEFAULT_PATH,
+                forceNew: true,
+                query: {
+                    password: `${SocketConfig.ADMIN_PASSWORD}`
+                }
+            });
+
+            let debateInfo = {
+                title: 'My new debate',
+                description: 'Test debate'
+            };
+            admin.emit("newDebate", debateInfo, (debateID) => {
+                id = debateID;
+                done();
+            });
+        });
+
+        beforeEach((done) => {
+            debate = debateManager.nspAdmin.getActiveDebate(id);
+
+            client = io.connect(`http://localhost:${SocketConfig.SOCKET_PORT}${SocketConfig.DEBATE_NAMESPACE_PREFIX}${id}`, {
+                path: SocketConfig.DEFAULT_PATH,
+                forceNew: true,
+            });
+
+            client.on('connect', () => {
+                done();
+            })
```

Comment:
End of instruction char ';' is missing.
```suggestion
            });
```